internal/pkg/errors/strings.go

```diff
@@ -39,12 +39,13 @@ const (
 	RestProxyNotAvailable = "Operation not supported: REST proxy is not available.\n"
 
 	// kafka topic commands
-	StartingProducerMsg      = "Starting Kafka Producer. Use Ctrl-C or Ctrl-D to exit."
-	StoppingConsumerMsg      = "Stopping Consumer."
-	StartingConsumerMsg      = "Starting Kafka Consumer. Use Ctrl-C to exit."
-	UpdateTopicConfigMsg     = "Updated the following configuration values for topic \"%s\":\n"
-	UpdateTopicConfigRestMsg = "Updated the following configuration values for topic \"%s\" (read-only configs were not updated):\n"
-	OmitTopicCountMsg        = "The topic count will be omitted as Kafka topics for this cluster could not be retrieved: %v"
+	StartingProducerMsg         = "Starting Kafka Producer. Use Ctrl-C or Ctrl-D to exit."
+	StoppingConsumerMsg         = "Stopping Consumer."
+	StartingConsumerMsg         = "Starting Kafka Consumer. Use Ctrl-C to exit."
+	UpdateTopicConfigMsg        = "Updated the following configuration values for topic \"%s\":\n"
+	UpdateTopicConfigRestMsg    = "Updated the following configuration values for topic \"%s\"%s:\n"
+	ReadOnlyConfigNotUpdatedMsg = "(read-only configs were not updated)"
```

Comment:
Some of these variables might make more sense to be moved to `internal/cmd/kafka` if they're only used there.

Reply:
(No reason for them to be public)